Polyhedron/demo/Polyhedron/Scene_polylines_item.cpp

```diff
@@ -34,6 +34,7 @@ Scene_polylines_item::create_Sphere(double R) const
 void
 Scene_polylines_item::initialize_buffers(CGAL::Three::Viewer_interface *viewer = 0) const
 {
+    QOpenGLShaderProgram *program;
```

Comment:
@maxGimeno : I do not understand why `program` was a data member of the class. It is sufficient to have a variable local to a few functions.